CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+* @stuhlmueller
+/notebooks/ @brachbach
+/tests/ @stuhlmueller
+/tests/test_scales.py @djthorne
+/tests/test_metaculus.py @brachbach
+/tests/test_logistic.py @djthorne
+/scripts/ @brachbach
+/ergo/scale.py @djthorne
+/ergo/theme.py @djthorne
+/ergo/platforms/ @stuhlmueller
```

Comment:
@stuhlmueller I did a bunch of the work on `/ergo/platforms/metaculus` and have some vague ideas for cleaning up

so I'd sort of like to be the owner

don't feel strongly though